### PR TITLE
feat: Adding some clarifications for Optimize usage with OpenSearch

### PR DESCRIPTION
**Pull request by grlimacan**
## Description

Adding some clarifications for Optimize usage with OpenSearch

closes https://github.com/camunda/camunda-optimize/issues/14043

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/setup/install.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ The existing Helm charts use the Elasticsearch configurations by default. The He
...
 
-For example, `CAMUNDA_OPERATE_ELASTICSEARCH_URL` becomes `CAMUNDA_OPERATE_OPENSEARCH_URL`.
+For example, `CAMUNDA_OPERATE_ELASTICSEARCH_URL` becomes `CAMUNDA_OPERATE_OPENSEARCH_URL`. In the case of Optimize, please make sure all variables have the proper `CAMUNDA_OPTIMIZE` prefix, i.e. `OPTIMIZE_ELASTICSEARCH_HTTP_PORT` becomes `CAMUNDA_OPTIMIZE_OPENSEARCH_HTTP_PORT`.
 
-Refer to the [Operate](/self-managed/operate-deployment/operate-configuration.md#settings-for-opensearch) and [Tasklist](/self-managed/tasklist-deployment/tasklist-configuration.md#elasticsearch-or-opensearch) configuration documentation for additional component configuration parameters to update.
+Refer to the [Operate](/self-managed/operate-deployment/operate-configuration.md#settings-for-opensearch), [Tasklist](/self-managed/tasklist-deployment/tasklist-configuration.md#elasticsearch-or-opensearch) and [Optimize](optimize/self-managed/optimize-deployment/configuration/system-configuration/#opensearch) configuration documentation for additional component configuration parameters to update.
```
Comment (PHWaechtler) [inline docs/self-managed/setup/install.md:48]: This Optimize link doesnt work (it resolves to
`https://preview.docs.camunda.cloud/pr-4430/docs/next/self-managed/setup/install/optimize/self-managed/optimize-deployment/configuration/system-configuration/#opensearch`)

File: versioned_docs/version-8.6/self-managed/setup/install.md [hunk 2]
```diff
@@ -41,11 +41,11 @@ The existing Helm charts use the Elasticsearch configurations by default. The He
...
 
-For example, `CAMUNDA_OPERATE_ELASTICSEARCH_URL` becomes `CAMUNDA_OPERATE_OPENSEARCH_URL`.
+For example, `CAMUNDA_OPERATE_ELASTICSEARCH_URL` becomes `CAMUNDA_OPERATE_OPENSEARCH_URL`. In the case of Optimize, please make sure all variables have the proper `CAMUNDA_OPTIMIZE` prefix, i.e. `OPTIMIZE_ELASTICSEARCH_HTTP_PORT` becomes `CAMUNDA_OPTIMIZE_OPENSEARCH_HTTP_PORT`.
 
-Refer to the [Operate](/self-managed/operate-deployment/operate-configuration.md#settings-for-opensearch) and [Tasklist](/self-managed/tasklist-deployment/tasklist-configuration.md#elasticsearch-or-opensearch) configuration documentation for additional component configuration parameters to update.
+Refer to the [Operate](/self-managed/operate-deployment/operate-configuration.md#settings-for-opensearch), [Tasklist](/self-managed/tasklist-deployment/tasklist-configuration.md#elasticsearch-or-opensearch) and [Optimize](optimize/self-managed/optimize-deployment/configuration/system-configuration/#opensearch) configuration documentation for additional component configuration parameters to update.
```
Comment (PHWaechtler) [inline versioned_docs/version-8.6/self-managed/setup/install.md:48]: The Optimize link doesnt work (it resolves to
`https://preview.docs.camunda.cloud/pr-4430/docs/next/self-managed/setup/install/optimize/self-managed/optimize-deployment/configuration/system-configuration/#opensearch`)

**Review by PHWaechtler — CHANGES_REQUESTED** [review]
Just one small suggestion and the link that currently doesnt resolve properly in the preview env

File: docs/self-managed/setup/install.md (hunk 1 above)
Comment (akeller) [reply docs/self-managed/setup/install.md:48]: https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#internal-links

> If the source is in docs and the target is in optimize, use the `$optimize$` token to prefix the URL.

File: versioned_docs/version-8.6/self-managed/setup/install.md (hunk 2 above)
Comment (grlimacan) [reply versioned_docs/version-8.6/self-managed/setup/install.md]: Thanks, corrected

**Comment by grlimacan** [comment]
Hi @akeller and @PHWaechtler , I have fixed the broken links and implemented the suggestions. During the process I also found an unrelated typo on the system-configuration.md files, so I used the opportunity to make that correct as well. Please re-review

File: versioned_docs/version-8.6/reference/supported-environments.md [hunk 3]
```diff
@@ -92,10 +92,12 @@ Requirements for the components can be seen below:
...
+| Optimize    | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.9+\*                                                                                                                                                                  |
 | Connectors  | OpenJDK 21+  |                                                                                                                                                                                                                   |
 | Web Modeler | -            | PostgreSQL 13.x, 14.x, 15.x, 16.x or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x, 16.x                                                                                                                              |
 
+\*Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are currently supported, please refer to the [Camunda 8](https://github.com/camunda/issues/issues/635) OpenSearch features.
```
Comment (conceptualshark) [inline versioned_docs/version-8.6/reference/supported-environments.md:99]: Does this need to be added to the /next version of the docs as well, or only 8.6?

**Review by PHWaechtler — APPROVED** [review]

File: versioned_docs/version-8.6/reference/supported-environments.md (hunk 3 above)
Comment (grlimacan) [reply versioned_docs/version-8.6/reference/supported-environments.md:99]: Only 8.6, with the next version we expect to support all features

**Pull request merged**